test/bats/helpers.bash

```diff
@@ -81,6 +81,16 @@ check_secret_deleted() {
   [[ "$result" -eq 0 ]]
 }
 
+archive_provider() {
+  if [[ -z "${ARTIFACTS}" ]]; then
+    return 0
+  fi
+
+  FILE_PREFIX=$(date +"%FT%H%M%S")
+
+  kubectl logs -l $1 --tail -1 -n kube-system > ${ARTIFACTS}/${FILE_PREFIX}-provider.logs
```

Comment:
The azure provider is installed in the `default` namespace. We should change the namespace in `azure.bats` to `kube-system`

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/651990f9d7a0f3a560eb2be254614bf8eac3a125/test/bats/azure.bats#L8